docs/rules/indent.md

```diff
@@ -523,6 +524,30 @@ var foo = { bar: 1,
             baz: 2 };
 ```
 
+### ImportDeclaration
+
+Examples of **incorrect** code for this rule with the `4, { ImportDeclaration: "first" }` option:
+
+```js
+/*eslint indent: ["error", 4, { ImportDeclaration: "first" }]*/
+
+import { foo,
+    bar,
+    baz,
+} from 'qux';
+```
+
+Examples of **correct** code for this rule with the `4, { ImportDeclaration: "first" }` option:
+
+```js
+/*eslint indent: ["error", 4, { ImportDeclaration: "first" }]*/
+
+import { foo,
+         bar,
+         baz,
+} from 'qux';
+```
```

Comment:
Can you add an example where the `ImportDeclaration` option is a number?

(At some point we should probably refactor these docs to avoid redundant examples for all of the "list" options, but for now it might be better to be thorough.)